paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -55,12 +63,13 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
 
     override val rootView: ViewGroup by lazy { viewBinding.root }
     override val bottomSheet: ViewGroup by lazy { viewBinding.bottomSheet }
-    override val linkAuthView: ComposeView by lazy { viewBinding.linkAuth }
-    override val scrollView: ScrollView by lazy { viewBinding.scrollView }
-    override val header: ComposeView by lazy { viewBinding.header }
-    override val fragmentContainerParent: ViewGroup by lazy { viewBinding.fragmentContainerParent }
-    override val notesView: ComposeView by lazy { viewBinding.notes }
-    override val bottomSpacer: View by lazy { viewBinding.bottomSpacer }
+
+    private val linkAuthView: ComposeView by lazy { viewBinding.linkAuth }
+    private val scrollView: ScrollView by lazy { viewBinding.scrollView }
+    private val header: ComposeView by lazy { viewBinding.header }
+    private val fragmentContainerParent: ViewGroup by lazy { viewBinding.fragmentContainerParent }
+    private val notesView: ComposeView by lazy { viewBinding.notes }
+    private val bottomSpacer: View by lazy { viewBinding.bottomSpacer }
```

Comment:
These need to be properties of `PaymentSheetActivity` until it’s migrated to Compose, too.